ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -312,11 +312,43 @@ def discrete_action_branches(self) -> Optional[Tuple[int, ...]]:
             return None
 
     def create_empty_action(self, n_agents: int) -> np.ndarray:
+        """
+        Generates a numpy array corresponding to an empty action (all zeros)
+        for a number of agents.
+        :param n_agents: The number of agents that will have actions generated
+        """
         if self.action_type == ActionType.DISCRETE:
             return np.zeros((n_agents, self.action_size), dtype=np.int32)
         else:
             return np.zeros((n_agents, self.action_size), dtype=np.float32)
 
+    def create_random_action(self, n_agents: int) -> np.ndarray:
+        """
+        Generates a numpy array corresponding to a random action (either discrete
```

Comment:
Add a note that this is used for example purposes and shouldn't be used in any serious code?

Reply:
You kind of sometimes want to generate random actions in serious code (epsilon greedy for example). Saying that it should not be used in serious code kind of makes me doubt how useful it will be

Reply:
Sorry, didn't realize there was a serious use for it (I was just going by your comment to Andrew that it was for simplifying example code).

If you think it's production quality, no need for the note.